### ISSUE TITLE
Comments being loaded into memory when using #Include

### ISSUE THREAD

**Issue by Animan8000** [Lexikos/AutoHotkey_L-Docs#530]
https://www.autohotkey.com/docs/Language.htm#comments

`Since comments are ignored when a script is launched, they do not impact performance or memory utilization.`

Doesn't seem to be entirely the case, at least with #Include. Didn't test much further at this point.

![image](https://user-images.githubusercontent.com/30600651/150674920-46a76dce-4821-46d8-97f9-d4869fc9925a.png)

Which is part of this included script:

![image](https://user-images.githubusercontent.com/30600651/150674941-028bf9ce-7b7f-4351-9659-5396eaad830e.png)

Basically one included file also uses an #Include to another script (library) but no idea if that's the cause or if comments are included already from the first included script.

**Comment by Lexikos** [comment]
What you've found is irrelevant. All of the script's text passes through memory while the script is being loaded. The program obviously can't know to skip it without first reading it from the file to know that it is there.

Temporary memory used during startup is "freed", which only means that it is handed back to the process heap and is free to be reallocated for some other purpose later. It does not mean that the previous content of the memory will be erased, which would take time, if not much.
